src/management/Akka.Management/AkkaManagementSettings.cs

```diff
@@ -65,9 +66,8 @@ public static Http Create(Config config)
             if (string.IsNullOrWhiteSpace(Hostname) || Hostname.Equals("<hostname>"))
             {
                 var addresses = Dns.GetHostAddresses(Dns.GetHostName());
-                Hostname = addresses
-                    .First(ip => !Equals(ip, IPAddress.Any) && !Equals(ip, IPAddress.IPv6Any))
-                    .ToString();
+                var ip = addresses.First(ip => !Equals(ip, IPAddress.Any) && !Equals(ip, IPAddress.IPv6Any));
+                Hostname = ip.AddressFamily == AddressFamily.InterNetworkV6 ? $"[{ip}]" : ip.ToString();
```

Comment:
IPAddress.ToString() returns "2001:db8:85a3:8d3:1319:8a2e:370:7348" instead of "[2001:db8:85a3:8d3:1319:8a2e:370:7348]". This can create problems when you string interpolate it into an URI